### PR TITLE
fix(logs): print the correct username and clientid

### PR DESCRIPTION
**Pull request by HJianBo**

### CODE REVIEW

**Review by zmstone — APPROVED** [review]
Would be great to have it explained why clientid and username from `mqtt_packet_connect` is not desired.

**Comment by HJianBo** [comment]
The `peer_cert_as_username` and `username_as_clientid` configuration options will assign a new username or clientid by the client’s certficate. Therefore, it is correct to use the username and clientid from Clientinfo.

**Comment by zmstone** [comment]
> The `peer_cert_as_username` and `username_as_clientid` configuration options will assign a new username or clientid by the client’s certificate. Therefore, it is correct to use the username and clientid from Clientinfo.

Thanks for the clarification.
For future reference: please include this into the commit log. Or add as comment here in github before hitting the Merge button so is becomes a part of the Merge commit log in git.

**Pull request merged**
